src/execution_plan/execution_plan_modify.c

```diff
@@ -128,13 +128,17 @@ void ExecutionPlan_RemoveOp(ExecutionPlan *plan, OpBase *op) {
 		// Remove new root's parent pointer.
```

Comment:
Just a side note,
what guarantee do we have that `op` has children?

Reply:
`assert(op->childCount == 1);`